modules/installation-options.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * architecture/architecture.adoc
+
+[id='installation-options-{context}']
+= Installation options
+
+{product-title} offers two installation options: fully-managed infrastructure and bring your own
+infrastructure. In version 4.0, the fully-managed option allows you to install a
+cluster in Amazon Web Services (AWS) that runs on Red Hat CoreOS nodes. If you want to
+use any other cloud or install your cluster on-premise, use the bring your own
```

Comment:
We need to call out that other clouds are coming soon.  This makes us look like we intend to stop at aws and won’t fly in perception.

Reply:
I could still use a external link to the roadmap, but I've added clarification that more support is coming.